### PR TITLE
Fix proxy container lint issue

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix proxy container lint issue

### Details and comments
The ubi images path has 3 slashes.

### CODE REVIEW

File: .github/scripts/dockerfile-security.rego [hunk 1]
```diff
@@ -25,7 +25,7 @@ deny[msg] {
 deny[msg] {
     input[i].Cmd == "from"
     val := split(input[i].Value[0], "/")
-    count(val) > 2
+    count(val) > 3
```
Comment (Tansito) [inline .github/scripts/dockerfile-security.rego:28]: Why are we changing this here @akihikokuroda ?

File: proxy/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi9/python-311
+FROM registry.access.redhat.com/ubi9/python-311@sha256:fae422b4e7dbc2e551919fb87b6004c06b731179be2ed2904427531a2d1ef3a2
```
Comment (Tansito) [inline proxy/Dockerfile:1]: do we want to start using `sha` from now on?

Comment (akihikokuroda) [reply proxy/Dockerfile:1]: I got docker file lint error llike this: `-:1 DL3006 warning: Always tag the version of an image explicitly`.  So I put the `sha`.

File: .github/scripts/dockerfile-security.rego (hunk 1 above)
Comment (akihikokuroda) [reply .github/scripts/dockerfile-security.rego:28]: It's a strange check.  The UBI image path is like this: `registry.access.redhat.com/ubi9/python-311` This path should be trusted but the lint fails.

Comment (psschwei) [reply .github/scripts/dockerfile-security.rego:28]: Yeah, I'm not sure how exactly the number of slashes corresponds to trustworthiness but que sera sera...

File: proxy/Dockerfile (hunk 2 above)
Comment (psschwei) [reply proxy/Dockerfile:1]: Do they also tag images or do we have to use the sha here? 
(I don't have a strong preference, just haven't worked with the UBI registry much before)

File: .github/scripts/dockerfile-security.rego (hunk 1 above)
Comment (Tansito) [reply .github/scripts/dockerfile-security.rego:28]: Haha understood 👍

File: proxy/Dockerfile (hunk 2 above)
Comment (Tansito) [reply proxy/Dockerfile:1]: yeah, me neither, that's why I'm asking, I'm totally fine with this approach if it has sense with what we are using

**Review by Tansito — APPROVED** [review]
LGTM, Aki 👍

File: proxy/Dockerfile (hunk 2 above)
Comment (akihikokuroda) [reply proxy/Dockerfile:1]: @psschwei They have tag like this `1-62.1717085982`.  It doesn't look user friendly, either.

**Review by psschwei — APPROVED** [review]

**Pull request merged**